packages/compartment-mapper/src/import.js

```diff
@@ -0,0 +1,51 @@
+// @ts-check
+
+import { defaultParserForLanguage } from './import-parsers.js';
+import {
+  loadLocation as loadLocationLite,
+  importLocation as importLocationLite,
+} from './import-lite.js';
+
+const { assign, create, freeze } = Object;
+
+/** @import {Application} from './types.js' */
+/** @import {ArchiveOptions} from './types.js' */
+/** @import {ExecuteOptions} from './types.js' */
+/** @import {ReadFn} from './types.js' */
+/** @import {ReadPowers} from './types.js' */
+
+// Add the default parserForLanguage option.
+const assignParserForLanguage = (options = {}) => {
```

Comment:
would it make sense to create a factory function for this?  idk how much DRY you're going for here.

Reply:
Meme with the normal curve where the developer is maximally DRY in the middle of their spiritual journey dot GIF.